webpack.config.js

```diff
@@ -16,5 +16,19 @@ module.exports = {
     new webpack.optimize.LimitChunkCountPlugin({
       maxChunks: 1
     })
-  ]
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.m?js$/,
+        exclude: /node_modules/,
+        use: {
```

Comment:
- [ ] It would be better if we ensure we use the content of the file given, kindly compare with [this](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)